spec/features/users/user_upgrade_request_spec.rb

```diff
@@ -1,6 +1,16 @@
 require "rails_helper"
 
 describe "Request an upgrade from trial user to editor", type: :feature do
+  before do
+    # prevent notify being called
```

Comment:
I don't _think_ we need to worry about preventing Notify being called... in our Rails test environment we use the `:test` mailer, not the Notify one:

https://github.com/alphagov/forms-admin/blob/7b09e0005c53242a17f30e207633ad9091be8b65/config/environments/test.rb#L41-L44

At least, that's what I've been relying on when I've been writing tests of sending mail in forms-runner 😅 

https://github.com/alphagov/forms-runner/blob/0fc0a5eee0d6f0bfa0e0e6db3005cf4d8eec9e82/spec/requests/forms/submit_answers_controller_spec.rb#L103-L120